src/cascadia/TerminalSettingsModel/KeyMapping.idl

```diff
@@ -54,7 +54,8 @@ namespace Microsoft.Terminal.Settings.Model
         BreakIntoDebugger,
         TogglePaneReadOnly,
         FindMatch,
-        NewWindow
+        NewWindow,
+        ToggleInboundPty

```

Comment:
I'll need to look into this more, but do we need this `ShortcutAction`? I was surprised to see `KeyMapping` was touched at all. If we do need it, mind adding an inline comment saying that this isn't a key-bindable action? Because it's so out of place haha.

Reply:
I think this was just the most expedient way of shooting something out of the args parser into the rest of the Terminal since it is already sending a bundle of actions that can be specified on the command line. I can add the comment.

Reply:
Also, I am not opposed to making it bindable to close off incoming connections and force a new window to spawn... but we will probably need to reconcile that with the multiprocess work after @zadjii-msft finishes landing that. So I'd rather not change the mechanism of action until we can better see the final outcome.

Reply:
I'd v. much rather find a way to do this that is NOT a binding at all. :/

Reply:
Took it out. Have a global state arg parse issue right now but I'll figure it out. It's on the todo.